### PR TITLE
Meg final design playground

### PR DESCRIPTION
**Pull request by MegEdnazednav**

### CODE REVIEW

File: components/PhoneTrackingButton.vue [hunk 1]
```diff
@@ -29,42 +32,53 @@ export default defineComponent({
...
-        navigator.geolocation.clearWatch(this.watcherId);
-        this.watcherId = -1;
-        this.$bvToast.toast("You are not being tracked anymore.", {
+      this.watcherId = navigator.geolocation.watchPosition((result) => {
+        if (!this.tracking) {
+          this.$bvToast.toast("You are being tracked.", {
             title: "Paradar message",
-            autoHideDelay: 3000,
-            variant: "success",
+            autoHideDelay: 200000,
```
Comment (JoshuaMoelans) [inline components/PhoneTrackingButton.vue:39]: Why show the toast for 3.3 minutes? It remains even after clicking the 'stop tracking' button.

**Review by JoshuaMoelans — CHANGES_REQUESTED** [review]
Just a small remark about the autoHideDelay, but style-wise it is looking real good 👌

File: components/PhoneTrackingButton.vue (hunk 1 above)
Comment (MegEdnazednav) [reply components/PhoneTrackingButton.vue]: Sorry forgot to put it back. Done now

Comment (JoshuaMoelans) [reply components/PhoneTrackingButton.vue]: 👍

**Review by JoshuaMoelans — APPROVED** [review]

**Pull request merged**